resources/assets/js/components/ApplicationReview/ApplicationReview.tsx

```diff
@@ -286,6 +286,17 @@ class ApplicationReview extends React.Component<
             >
               {application.applicant.user.email}
             </a>
+            {/* This span only shown for priority applicants */}
+            {application.applicant.user.is_priority && (
+              <span className="priority-status">
+                <i
+                  aria-hidden="true"
+                  className="fab fa-product-hunt"
+                  title="The Talent Cloud priority icon"
+                />
+                <span>Priority</span>
```

Comment:
Little late on this, but I guess this should be localized now that we merged in the react-intl branch?